google/cloud/storage/blob.py

```diff
@@ -193,6 +194,24 @@ def __init__(
         if generation is not None:
             self._properties["generation"] = generation
 
+    @property
+    def name(self):
+        """ Get the blob's name.
+
+        :rtype: str
+        :returns: The name for the blob object.
+        """
+        return self._name
+
+    @name.setter
```

Comment:
Recommend removing setter.

Reply:
@frankyn  If i remove setter then it throws an error like `AttributeError: can't set attribute`

Reply:
Is this why you recommended only adding pydocs instead? If so, then let's do that instead.

Reply:
@frankyn  Yes , that's why i recommend doc static doc string. i will update the PR. Thanks.